Docs/Instruments/4-Lane Drums/PS3 and Wii.md

```diff
@@ -27,6 +27,15 @@ Wii:
   - RB3:
 - Device name:
 
+Santroller:
+- Vendor/product ID:
+  - `1209:2882`
+- Revision:
+  - `0x091x`
+  - The last nibble may vary, see [the main Santroller doc](../../Other/Santroller.md).
+- Device name:
+  - Santroller
```

Comment:
Preferably, properties with only one possible value should be on a single line. Also, the device name should be put in a single-line code block.

```suggestion
Santroller:
- Vendor/product ID: `1209:2882`
- Revision: `0x091x`
  - The last nibble may vary, see [the main Santroller doc](../../Other/Santroller.md).
- Device name: `Santroller`
```